substratest/settings.py

```diff
@@ -13,13 +13,11 @@
 _DEFAULT_NETWORK_CONFIGURATION_PATH = os.path.join(_CURRENT_DIR, "../", "values.yaml")
 _SUBSTRA_TESTS_CONFIG_FILEPATH = os.getenv("SUBSTRA_TESTS_CONFIG_FILEPATH", _DEFAULT_NETWORK_CONFIGURATION_PATH)
 
-_DEFAULT_SUBSTRA_TOOLS_TAG_LOCAL = (
-    f"latest-nvidiacuda11.8.0-base-ubuntu22.04-python{sys.version_info.major}.{sys.version_info.minor}"
-)
-_DEFAULT_SUBSTRA_TOOLS_TAG_REMOTE = "latest"
+_DEFAULT_SUBSTRA_TOOLS_TAG_LOCAL = f"{sys.version_info.major}.{sys.version_info.minor}-slim"
+_DEFAULT_SUBSTRA_TOOLS_TAG_REMOTE = "3.12-slim"
 
-_DEFAULT_SUBSTRA_TOOLS_IMAGE_REMOTE = f"substra/substra-tools:{_DEFAULT_SUBSTRA_TOOLS_TAG_REMOTE}"
-_DEFAULT_SUBSTRA_TOOLS_IMAGE_LOCAL = f"ghcr.io/substra/substra-tools:{_DEFAULT_SUBSTRA_TOOLS_TAG_LOCAL}"
+_DEFAULT_SUBSTRA_TOOLS_IMAGE_REMOTE = f"python:{_DEFAULT_SUBSTRA_TOOLS_TAG_REMOTE}"
+_DEFAULT_SUBSTRA_TOOLS_IMAGE_LOCAL = f"python:{_DEFAULT_SUBSTRA_TOOLS_TAG_LOCAL}"
```

Comment:
A renaming of the variable `_DEFAULT_SUBSTRA_TOOLS_IMAGE_REMOTE` to smtgh like `_DEFAULT_DOCKER_IMAGE_REMOTE` would be great I think :)